articles/ToDoList-with-Flask-Firebase/index.md

```diff
@@ -0,0 +1,241 @@
+
+
+# Building a To-Do List Web App with Flask and Firebase
+
+ 
+
+### Introduction
+
+Flask is microweb framework in Python, which is light-weight and easy to use. Firebase Realtime Database is a powerful no-SQL cloud database with SDKs, and client libraries in many commonly used languages including Python. Together, both of them can be used to build dynamic and flexible web applications quickly. Let’s dive right in and learn how to use these technologies together by building a To-Do List web app! 
+
+### Prerequisites
+
+- Basic knowledge of Python and HTML.
```

Comment:
Moving forward, avoid the word "basic". If someone finds it difficult or is unable to follow through with your article they might be discouraged because the knowledge needed was termed as basic.